### PR TITLE
Fix semantic conventions for PG and PG Pool

### LINKED ISSUE #116: [pg + pg-pool] Update DB semantic conventions
https://github.com/open-telemetry/opentelemetry-specification/pull/575

### PR DESCRIPTION
**Pull request by svrnm**
## Which problem is this PR solving?
This should fix #116, #296, #297 and #465

## Short description of the changes
Replace all official semantic conventions with their values from @opentelemetry/semantic-conventions package. 
Leave some of the old custom attributes as they are, but put them into a namespace db.postgresql (while not officialy stated in https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md) there are some other examples with db.<db.system>, e.g. db.mssql.instance_name or db.mongodb.collection

```
  PG_VALUES = 'db.postgresql.values',
  PG_PLAN = 'db.postgresql.plan',
  IDLE_TIMEOUT_MILLIS = 'db.postgresql.idle.timeout.millis',
  MAX_CLIENT = 'db.postgresql.max.client',
```

### CODE REVIEW

File: examples/postgres/package.json [hunk 1]
```diff
@@ -35,11 +35,11 @@
     "@opentelemetry/exporter-zipkin": "^0.19.0",
     "@opentelemetry/instrumentation": "^0.19.0",
     "@opentelemetry/instrumentation-http": "^0.19.0",
-    "@opentelemetry/instrumentation-pg": "^0.15.0",
+    "@opentelemetry/instrumentation-pg": "file:///Users/neumanns/Projects/opentelemetry-js-contrib/plugins/node/opentelemetry-instrumentation-pg",
```
Comment (dyladan) [inline examples/postgres/package.json:38]: Please undo this

File: plugins/node/opentelemetry-instrumentation-pg/src/enums.ts [hunk 2]
```diff
@@ -14,27 +14,10 @@
...
-  PEER_SERVICE = 'peer.service',
-
-  // PG specific -- not specified by spec
-  PG_VALUES = 'pg.values',
-  PG_PLAN = 'pg.plan',
-
-  // PG-POOL specific -- not specified by spec
-  IDLE_TIMEOUT_MILLIS = 'idle.timeout.millis',
-  MAX_CLIENT = 'max',
+  PG_VALUES = 'db.postgresql.values',
```
Comment (dyladan) [inline plugins/node/opentelemetry-instrumentation-pg/src/enums.ts:19]: I think this one might actually be in semconv?

**Review by dyladan — CHANGES_REQUESTED** [review]
Blocked because your local path is in the example package.json

File: examples/postgres/package.json (hunk 1 above)
Comment (svrnm) [reply examples/postgres/package.json]: 🤦‍♂️ ... good catch. I changed that back.

File: plugins/node/opentelemetry-instrumentation-pg/src/enums.ts (hunk 2 above)
Comment (svrnm) [reply plugins/node/opentelemetry-instrumentation-pg/src/enums.ts:19]: No, there is none for storing the "values"

**Review by obecny — APPROVED** [review]
lgtm

File: plugins/node/opentelemetry-instrumentation-pg/src/enums.ts (hunk 2 above)
Comment (svrnm) [reply plugins/node/opentelemetry-instrumentation-pg/src/enums.ts:19]: will keep this for now, maybe spec needs to add something here.

File: examples/postgres/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (svrnm) [reply examples/postgres/package.json]: fixed

**Review by vmarchaud — APPROVED** [review]

**Pull request merged**